src/pretix/base/migrations/0251_logentry_organizer.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.2.4 on 2023-11-20 12:38
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.db.models import F, Subquery, OuterRef
+
+
+def backfill_organizer(apps, schema_editor):
+    LogEntry = apps.get_model("pretixbase", "LogEntry")
+    Event = apps.get_model("pretixbase", "Event")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    LogEntry.objects.filter(
+        organizer_link__isnull=True, event__isnull=False
+    ).update(organizer_link_id=Subquery(
+            Event.objects.filter(pk=OuterRef('event_id')).values('organizer_id'),
+        )
+    )
+    for ct in ContentType.objects.all():
+        try:
+            model = apps.get_model(ct.app_label, ct.model)
+        except LookupError:
+            continue
+        if "organizer" in model._meta.fields:
+            LogEntry.objects.filter(
+                organizer_link__isnull=True, event__isnull=False, content_type=ct,
+            ).update(
+                organizer_link_id=Subquery(model.objects.filter(pk=OuterRef('object_id')).values('organizer_id'))
+            )
+        elif "event" in model._meta.fields:
+            LogEntry.objects.filter(
+                organizer_link__isnull=True, event__isnull=False, content_type=ct,
+            ).update(
+                organizer_link_id=Subquery(model.objects.filter(pk=OuterRef('object_id')).values('event__organizer_id'))
+            )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("pretixbase", "0250_eventmetaproperty_filter_public"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="logentry",
+            name="organizer_link",
```

Comment:
What is the reason you called this `organizer_link` instead of just `organizer`?

Reply:
Because we already have a `def organizer` and it wouldn't be backwards-compatible otherwise. We can rename it once we migrated all data on production, if we want to

Reply:
Ah, thanks for explaining this!